test/modules/core/lib/deck.spec.js

```diff
@@ -97,3 +97,73 @@ test('Deck#rendering, picking, logging', t => {
     }
   });
 });
+
+test('Deck#auto view state', t => {
```

Comment:
This looks good. would it make sense to have some test case with "embedded/nested view states"?

Reply:
There are `ViewManager` tests that check the handling of various `viewState` formats. This test focuses on the internal state, which is always saved as an nested view state map.